nvdaHelper/UIARemote/UIARemote.cpp

```diff
@@ -104,9 +115,12 @@ extern "C" __declspec(dllexport) bool __stdcall msWord_getCustomAttributeValue(I
 			LOG_DEBUG(L"Extension not supported");
 		}
 	} catch (std::exception& e) {
-		std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
-		auto what = converter.from_bytes(e.what());
-		LOG_ERROR(L"msWord_getCustomAttributeValue exception: "<<what);
+		const std::string what{e.what()};
+		// convert exception message to unicode
+		int wideLen = MultiByteToWideChar(CP_UTF8, 0, what.c_str(), what.length(), nullptr, 0);
+		auto wideBuf = std::make_unique<wchar_t[]>(wideLen);
+		MultiByteToWideChar(CP_UTF8, 0, what.c_str(), what.length(), wideBuf.get(), wideLen);
+	LOG_ERROR(L"msWord_getCustomAttributeValue exception: "<<(wideBuf.get()));

```

Comment:
This seems like something that belongs in a utility function. It is hard to determine correctness at a glance, and easy to make a mistake. 

I think there is a chance that `wideBuf ` is may not be null terminated here.  Using `MultiByteToWideChar` will include return a size which includes the null termination if `cbMultiByte` is `-1`, however this uses the `size()` from `basic_string` which does not count the null terminating character.

An alternative to explicitly dynamically allocating `wchar` array to store the converted string is to use a `std::vector<wchar_t`. One advantage of the vector approach is the easy of initializing the memory eg:
``` c++
std::vector<wchar_t> wideBuf (wideLen, L'\0');
```
This however sacrifices speed for correctness, given this is exception handling I'd say it is a fair tradeoff.

To keep the current heap array approach, just add 1 to `wideLen` when allocating, then after the second call to `MultiByteToWideChar` set `wideBuf[wideLen]=L'\0'`

```suggestion
		auto wideBuf = std::make_unique<wchar_t[]>(1 + wideLen); // add space for a null char.
		MultiByteToWideChar(CP_UTF8, 0, what.c_str(), what.length(), wideBuf.get(), wideLen);
		wideBuf[wideLen] = L'\0';
		LOG_ERROR(L"msWord_getCustomAttributeValue exception: "<<(wideBuf.get()));
```